src/templates/bash-template

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright 2020 AstroLab Software
+# Author: Yash Datta
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+if [ $# -le 1 ]; then
+  echo "Usage: --config <config-file>"
+  exit 1
+fi
+
+VARS=$@
+
+# Parse config and rules file paths
+while test $# -gt 0
+do
+  case "$1" in
+    --config)
+      shift
+      CONF_FILE_PATH=$1;;
+  esac
+  shift
+done
+
+# Convert file paths to their basenames, since spark just needs base names to refer to these files
+CONF_FILE=$(basename $CONF_FILE_PATH)
+
+VARS=${VARS/$CONF_FILE_PATH/$CONF_FILE}
+
+lib_dir=$(cd "$(dirname ${BASH_SOURCE[0]})"/..; pwd)
+
+${{template_declares}}
+
+KRB5_CONF=""
+
+LOG_FILE=$lib_dir/run.out
+
+echo "Running spark-submit..."
+
+spark-submit \
+--master ${SPARK_MASTER} \
+--deploy-mode cluster \
+--driver-memory 8g \
+--driver-cores 4 \
+--num-executors 5 \
+--executor-memory 2g \
+--executor-cores 4 \
```

Comment:
All those options above are rather specific. Could it be adapted depending on the environment we deploy? A minima, we should have a local[*] case to allow testing. Otherwise on production, we currently use mesos.

Reply:
Yes but then we have to specify the environment while creating the package.
Hmm, let me think what can be done

Reply:
As discussed, there should be only one package, with different run commands (or scripts, like `start_prod`, `start_local`, ...)